packages/mui-base/src/useMenu/useMenu.types.ts

```diff
@@ -4,6 +4,14 @@ import { MenuItemMetadata } from '../useMenuItem';
 import { MenuProviderValue } from './MenuProvider';
 
 export interface UseMenuParameters {
+  /**
+   * If `true` (Default) will focus the `[role="menu"]` if no focusable child is found. Disabled
```

Comment:
This description is not exactly correct:
- _will focus the `[role="menu"]` if no focusable child is found_ - I'm not aware of any logic we have that does this
- _disabled children are not focusable_ - they are
- _if you set this prop to `false` focus will be placed on the parent modal container_ - if the prop is false, we just don't set focus at all, do we?

Reply:
I copied the description from Material UI, I will revise it to make it compatible with Base UI's default implementation.